inference-engine/src/mkldnn_plugin/nodes/mkldnn_input_node.h

```diff
@@ -21,19 +21,18 @@ class MKLDNNInputNode : public MKLDNNNode {
     void createPrimitive() override;
     bool created() const override;
 
-    void execute(mkldnn::stream strm) override;
-    void withMeanImage() {
-        isMeanImage = true;
-    }
+    void withMeanImage();
+    const InferenceEngine::Blob::CPtr getConstBlob() const;
+    MKLDNNMemoryPtr getMemoryPtr() const;
```

Comment:
Shouldn't it be const MKLDNNMemoryPtr?

Reply:
I think here it isn't mandatory. The const MKLDNNMemoryPtr is just prevent assign to another share pointer.